### PR TITLE
Set .setup_dev.sh script to be executable, and readme to tell folks t…

### PR DESCRIPTION
**Pull request by mtauraso**
…o run it

## Checklist

- [X] This PR is meant for the `lincc-frameworks/python-project-template` repo and not a downstream one instead.
- [X] This change is linked to an open issue
- [ ] This change includes integration testing, or is small enough to be covered by existing tests

### CODE REVIEW

**Review by drewoldag — APPROVED** [review]
Looks good, once that comment is resolved. :)

File: python-project-template/README.md.jinja [hunk 1]
```diff
@@ -37,7 +37,7 @@ Once you have created a new environment, you can install this project for local
 development using the following commands:
 
 ```
->> pip install -e .'[dev]'
+>> ./.setup_dev.sh
 >> pre-commit install
```
Comment (delucchi-cmu) [inline python-project-template/README.md.jinja:41]: Can remove this line, as `pre-commit install` is run in `/.setup_dev.sh`.

File: python-project-template/README.md.jinja [hunk 2]
```diff
@@ -37,7 +37,7 @@ Once you have created a new environment, you can install this project for local
 development using the following commands:
 
 ```
->> pip install -e .'[dev]'
+>> ./.setup_dev.sh
```
[the remark this answers is not in the document]
Comment (mtauraso) [reply python-project-template/README.md.jinja:40]: Removed in most recent version.

File: python-project-template/README.md.jinja (hunk 1 above)
Comment (mtauraso) [reply python-project-template/README.md.jinja]: Removed this, and altered the note about pre-commit install below.

**Pull request merged**